src/prefect/agent/agent.py

```diff
@@ -80,6 +91,7 @@ def __init__(
         labels: Iterable[str] = None,
         env_vars: dict = None,
         max_polls: int = None,
+        api_address: str = None,
```

Comment:
Yeah this could be a bit confusing and possible to be mistaken with the actual cloud/server API address. Maybe more along the lines of `agent_address` or something